spec/requests/customers_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+RSpec.describe "CustomersControllers", type: :request do
```

Comment:
You didn't do requests tests. 5 tests are still pending and one is failed in `customers_controller`.